libs/cart/state/testing/src/factories/stateful-configurable-cart-item.factory.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+
+import { DaffModelFactory } from '@daffodil/core/testing';
+import { DaffMockConfigurableCartItem } from '@daffodil/cart/testing';
+import { DaffStatefulCartItem, DaffCartItemStateEnum, DaffStatefulConfigurableCartItem } from '@daffodil/cart/state';
+
+export class DaffMockStatefulConfigurableCartItem extends DaffMockConfigurableCartItem implements DaffStatefulCartItem {
```

Comment:
Should implement `DaffStatefulConfigurableCartItem`

Reply:
fixed